documentation/staging/content/quickstart/create-domain.md

```diff
@@ -24,7 +24,9 @@ weight: 6
 
 1.	Create a new image with a domain home by running the [create-domain](http://github.com/oracle/weblogic-kubernetes-operator/blob/main/kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/create-domain.sh) script.
 
-    {{% notice tip %}} For a detailed understanding of the steps that the `create-domain.sh` script performs for you, see the bulleted items under [Use the script to create a domain]({{< relref "/samples/simple/domains/domain-home-in-image/#use-the-script-to-create-a-domain" >}}).
+    The script's behavior is controlled by an inputs YAML file plus command-line options. The script downloads the [WebLogic Image Tool](https://oracle.github.io/weblogic-image-tool/) and [WebLogic Deploy Tool](https://oracle.github.io/weblogic-deploy-tooling/), and uses these tools to create a new image with a domain home, a domain resource YAML file that references the image, and, if the `-e` option is specified, deploy the domain resource to Kubernetes. For a detailed understanding of the steps that the `create-domain.sh` script performs for you, see the bulleted items under [Use the script to create a domain]({{< relref "/samples/simple/domains/domain-home-in-image/#use-the-script-to-create-a-domain" >}})
```

Comment:
Long story short, please change "inputs YAML file" to "inputs file".

The inputs file technically happens to be YAML but I don't think we want to emphasize the fact. 

The long story:  IMO, this fosters "conflation confusion" with the domain resource file and other k8s resources -- which are also usually YAML. It's also a little confusing because teh inputs file is better understood as a properties file: it doesn't have a nested structured schema that's typical of YAML files - it's just a flat set of name/value pairs.

Reply:
Will do.